src/settings/test.rs

```diff
@@ -64,7 +64,7 @@ fn env_vars_take_precedence() {
         "FXA_EMAIL_AWS_SQSURLS_COMPLAINT",
         "FXA_EMAIL_AWS_SQSURLS_DELIVERY",
         "FXA_EMAIL_AWS_SQSURLS_NOTIFICATION",
-        "FXA_EMAIL_BOUNCELIMITS_ENABLED",
+        "FXA_EMAIL_DELIVERYPROBLEMLIMITS_ENABLED",
```

Comment:
need to make sure to document this change in the deploy doc

Reply:
Updated!